### PR TITLE
2022 10 02 draft review

### PR DESCRIPTION
**Pull request by xyzrus**

### CODE REVIEW

File: _posts/randy/2022-09-10-tech-juggling-reality.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
```
Comment (adamjberg) [inline _posts/randy/2022-09-10-tech-juggling-reality.md]: This is an old article right?  Shouldn't be showing up here

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+<audio controls="controls">
+  <source type="audio/mp3" src="{{"/assets/audio/all-creative-now.mp3" | relative_url}}"></source>
+  <p>Your browser does not support the audio element.</p>
+</audio>
+
+      Audio - "Jumping Off Point" 2:37
+
+## DRAWING THE LINES
+
+In the beginning, there was only one creator – woever, whatever dialed in the FORCE that energized the universe. Long story short, there came a reason for being. A little later, CREATIVE TYPES of human debuted with [cave paintings](https://www.atlasobscura.com/articles/cosquer-cave-art-underwater), leaving messages on walls that lasted longer than their being. Some of these priceless artifacts will outlive us all, but none will survive to the end of the universe.
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md]: is woever intentional?

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+Now that we are all CREATORS, how is this going to work? Not long ago everyone went off to pay for university degrees in business rape-and pillage, North Elbonian history and basket weaving – leaving no one behind to be doctors, lawyers, police officers, firefighters, mechanics, farmers and teachers. If everyone is a creator, who is going to do the hunting and gathering? How do we eat?
+
+I am not old enough to have grade-school grandchildren, but they both told us recently they want to be [YouTubers](https://www.creatorhandbook.net/how-to-become-a-youtuber-all-the-required-steps) – wannabe content producers. Kids see adult technology and assume it was made for them, because they already know how to use it better. Arrived in the world with phones in their face, they grow up with a fixed arm's length relationship with apples and androids, and only instant gratification will do until death.
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md:48]: arms without ' I think?

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+I was mostly concerned with hopelessly fixing my dirt-jumped 10-speed bicycle at their age, but it looks like we've already come full creative circle in the family. The kids' parents are not officially artists, but they are fully immersed in CREATING CREATIVE tools used to help other artists with business management software [Dubsado](www.dubsado.com). Lots of molecules rubbed off that branch.
+
+## NOT YOUR-TUBE: QUALITY of PRODUCT vs. QUANTITY of REVENUE
+
+As with any success curve, highly successful CREATORS are rare  but soak up an overabundance of revenue. There are many good "YouTubers" (97% YT earn <minimum wage) but a few have figured out how to game the system. It is all about the "algorithm" - the secret recommendation formula which is rarely even understand by the algorithmers, let alone algorithmees.
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md]: perhaps spell out less than or just space after <

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+Google/Facebook/Apple/Etc is in the business of making money – and they have to make all the money or they take their marbles and go to the next hustle. They do not care twit if content is good; they just care if you pay for play or keep clicking on the boxed shit they put in front of your face. Free is fantastic, but if you are not a paying customer, you are the product being sold. All the ads mindlessly endured in dead ends make for a lot of billionaires at the head end.
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md]: twit?

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+Throwing around pretend money (so they can pretend they are a player) is the high-tech corporate version of socialism – a bribe offered to people with the money made off same peoples' backs. $250 million can't buy you one professional baseball player - maybe we are all playing the wrong game.
+
+CREATIVITY, however difficult - has never been so popular, both from a consumption and a delivery standpoint. While there are many more opportunities than ever before, these new digital markets are overwhelmed with many more micro-competitors.
+
+Flip through the online design for hire site [fiverr](www.fiverr.com) for fiverr minutes be dually amazed: 1) there are so many custom project designers for hire so cheaply, and 2) you have been replaced by the internet and have no chance. Why would anyone pay thousands to you face when they can pay hundreds to someone far away? On the other hand, how does anyone that wants to get rich get there at $5 or $100 and up?
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md]: you -> your

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+There will be various descriptions of the coming ages, eras or economies since the Information Age is old aged by now. A new BUZZWORD developing is in recognition of the expected CREATIVE ECONOMY. Already 25% of the US workforce toils in what is considered a CREATIVE ROLE.
+
+## TURN ON, TUNE IN – DON'T DROP OUT
+
+Our youth are not shy and will do very well with phones on face and faces on phone. Hopefully they will discover phones can also take some pictures and videos horizontally, where viewers can see fewer skies and bellies, the whole group in the "selfie" shot, and wide format landscapes. We can say it was their invention - we can be dazzled as they rotate +/- 90 degrees.
+
+Rather than be turned away by naysayers, people of all ages should embrace the technologies in our hands and in our faces. Accept your fate, and turn on your devices – get ready to learn. It will take days, months and years to figure out the "RITHYM" and the "RHYME" of your CREATIVE STREAK, but you have a few. Do not worry that you will fall behind, as that already happened again while his sentence was being  composed.
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md]: Is the RITHYM spelling intentional?

File: _posts/randy/2022-10-02-all-creative-now.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+The best players at any game are the most talented players that never give up. If all odds are against the SELF-STARTER, the payoff for a win is going to be pretty big. The attraction to be a CREATOR, an INFLUENCER, a CREATOR, may secretly be a FORCE within a lot of us.
+
+...
+
+Randy Berg
+xyz digital inc.
+
+...
+
+## NAB AMPLIFY – AMPLIFICATION
```
Comment (adamjberg) [inline _posts/randy/2022-10-02-all-creative-now.md]: Does this intentionally come after signature?

Comment (adamjberg) [reply _posts/randy/2022-10-02-all-creative-now.md]: Maybe I'm just tired, but this feels out of place

**Review by adamjberg — CHANGES_REQUESTED** [review]
Excellent article.  Few more typos (some maybe intentional) than usual.

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 5 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: what a twitter is

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 4 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: <ok>

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 3 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md:48]: possessive's

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 2 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: woenever

File: _posts/randy/2022-09-10-tech-juggling-reality.md (hunk 1 above)
Comment (xyzrus) [reply _posts/randy/2022-09-10-tech-juggling-reality.md:1]: don't know what

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 4 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: -<

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 6 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: bad proofreader

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 7 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: algo 'Rithym

File: _posts/randy/2022-10-02-all-creative-now.md (hunk 8 above)
Comment (xyzrus) [reply _posts/randy/2022-10-02-all-creative-now.md]: was intentional - block from one source; moved to body

File: _posts/randy/2022-09-10-tech-juggling-reality.md (hunk 1 above)
Comment (xyzrus) [reply _posts/randy/2022-09-10-tech-juggling-reality.md]: there was an overdue typo fix i think

**Comment by xyzrus** [comment]
done for now

**Review by adamjberg — APPROVED** [review]

**Pull request merged**